src/components/split-button/split-button.style.ts

```diff
@@ -16,9 +16,21 @@ const StyledSplitButton = styled.div`
   & > ${StyledButton} {
     margin: 0;
     &:focus {
-      border: 3px solid var(--colorsSemanticFocus500);
-      outline: none;
-      margin: -1px;
+      ${({ theme }) => css`
+        ${theme.focusRedesignOptOut &&
```

Comment:
**suggestion** (non-blocking): as elsewhere, I'd personally prefer to write this as a ternary.

Separately, I'm surprised the old styles haven't been extracted as `oldFocusStyles` or a similar const. I don't much mind personally but it seems inconsistent to do this in most components but not here.

Reply:
The suggestion is valid I just don't think this is super important for the legacy stuff so leaving it as is